### PR TITLE
Add actionCartGetPackageShippingCost hook

### PR DESCRIPTION
**Pull request by PrestaEdit**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Add related hook correspond to https://github.com/PrestaShop/PrestaShop/pull/29000
| Type?             | improvement
| BC breaks?        |no
| Deprecations?     | no

### CODE REVIEW

**Review by Hlavtox — APPROVED** [review]

**Comment by hibatallahAouadni** [comment]
We'll test with this PR https://github.com/PrestaShop/PrestaShop/pull/29000 when it has `Waiting for QA` label on it :wink: 
Please remove the label Blocked when the reltaed PR get `Waiting for QA`.

Thanks in advance :pray:

**Comment by kpodemski** [comment]
This will be in 9.0.0.sql

**Comment by hibatallahAouadni** [comment]
@PrestaEdit Could you please resolve the conflicts 🙏

**Comment by kpodemski** [comment]
I'm adding blocked label because PR from the core waits for QA.

File: upgrade/sql/8.1.0.sql [hunk 1]
```diff
@@ -35,4 +35,4 @@ CREATE TABLE `PREFIX_product_attribute_lang` (
 INSERT INTO `PREFIX_configuration` (`name`, `value`, `date_add`, `date_upd`) VALUES  
   ('PS_PRODUCT_REDIRECTION_DEFAULT', '404', NOW(), NOW());
 UPDATE `PREFIX_product` SET `redirect_type` = 'default' WHERE `redirect_type` = '404';
-UPDATE `PREFIX_product_shop` SET `redirect_type` = 'default' WHERE `redirect_type` = '404';
+UPDATE `PREFIX_product_shop` SET `redirect_type` = 'default' WHERE `redirect_type` = '404';
```
Comment (PrestaEdit) [inline upgrade/sql/8.1.0.sql:38]:
```suggestion
UPDATE `PREFIX_product_shop` SET `redirect_type` = 'default' WHERE `redirect_type` = '404';

```

File: upgrade/sql/9.0.0.sql [hunk 2]
```diff
@@ -0,0 +1,5 @@
+SET SESSION sql_mode='';
+SET NAMES 'utf8mb4';
+
+INSERT IGNORE INTO `PREFIX_hook` (`id_hook`, `name`, `title`, `description`, `position`) VALUES
+  (NULL, 'actionCartGetPackageShippingCost', 'After getting package shipping cost value', 'This hook is called in order to allow to modify package shipping cost', '1');
```
Comment (PrestaEdit) [inline upgrade/sql/9.0.0.sql:5]:
```suggestion
  (NULL, 'actionCartGetPackageShippingCost', 'After getting package shipping cost value', 'This hook is called in order to allow to modify package shipping cost', '1');

```

**Comment by PrestaEdit** [comment]
The main PR is merged, blocking label is not true anymore.

**Review by kpodemski — APPROVED** [review]

**Comment by Hlavtox** [comment]
@AureRita There is no conflict, this PR adds one new single file.

**Review by paulnoelcholot — APPROVED** [review]

**Pull request merged**